src/java.desktop/macosx/classes/com/apple/laf/AquaButtonUI.java

```diff
@@ -217,10 +221,15 @@ protected void uninstallKeyboardActions(final AbstractButton b) {
     }
 
     protected void uninstallListeners(final AbstractButton b) {
-        super.uninstallListeners(b);
         final AquaButtonListener listener = (AquaButtonListener)b.getClientProperty(this);
         b.putClientProperty(this, null);
         if (listener != null) {
+            b.removeMouseListener(listener);
+            b.removeMouseListener(listener);
```

Comment:
Why are we removing same listener twice?

Reply:
I just rolled back the changes I did in this file in JDK-8249548. Looks like this was has been there for some time. See the file in commit made in this file before JDK-8249548 https://github.com/openjdk/jdk/blob/c18080fef714949221c8ddabc4e23d409575c3d3/src/java.desktop/macosx/classes/com/apple/laf/AquaButtonUI.java. 
But yes, it should not be done and I will remove it and update the PR.